### PR TITLE
{tools}[GCCcore/10.2.0] OpenStackClient v5.5.0 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by ThomasHoffmann77**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/o/openstackclient/openstackclient-5.5.0-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,133 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2021/08
+easyblock = 'PythonBundle'
+
+name = 'openstackclient'
```
Comment (branfosj) [inline easybuild/easyconfigs/o/openstackclient/openstackclient-5.5.0-GCCcore-10.2.0.eb:4]: Please use `OpenStackClient` for the name.

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/o/openstackclient/openstackclient-5.5.0-GCCcore-10.2.0.eb (hunk 1 above)
Comment (ThomasHoffmann77) [reply easybuild/easyconfigs/o/openstackclient/openstackclient-5.5.0-GCCcore-10.2.0.eb]: @branfosj done

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.5
See https://gist.github.com/c5e926a5d29a32eecf5ebe46db43f724 for a full test report.

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/aa6941d55e305434ae560ca27c0fde9f for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @ThomasHoffmann77!

**Pull request merged**